packages/components/modules/activity-log/web/ActivityLogComponent/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const getUpdateMessage = (verb: string, diff: Record<string, any>) => {
+  if (!diff) return `Updated ${verb.split('.')[0]}`
+
+  const updates: string[] = []
+
+  Object.keys(diff).forEach((key) => {
```

Comment:
Can we improve this to make it more readable:

```
const updateMessages: Record<keyof LogDiff, string> = {
  image: 'Updated their profile picture',
  banner_image: 'Updated their profile banner',
  biography: 'Updated their bio',
};

Object.keys(diff).forEach((key) => {
  const message = updateMessages[key as keyof LogDiff] || `Updated ${key.replace('_', ' ')}`;
  updates.push(message);
});
```

Reply:
Ty for the comments, indeed we can make it easier.
@priscilladeroode please could you tell me if it is more readable, now?